### PR TITLE
1090 customize dropdown menu

### LINKED ISSUE #1090: Customize dropdown menu
Customize the dropdown menu in catalog.
The contributions/{contributions_id}/review page has a dropdown menu which should be customized to match the zeplin design.
Check if bootstrap select and option html tag can be customized.
An example implementation is here (https://www.w3schools.com/howto/howto_custom_select.asp)

### PR DESCRIPTION
**Pull request by minump**
## Description
The arrow in the dropdown menu is changed. Dropdown menu is found in contributions/review page.
The orange caret down arrow and the orange underline for the select html tag is implemented.

Tried the Saas variables and other select & option tag css properties.

The javascript needed for the full customization is commented out. The other related styling classes is also present in the style.css file. These can be deleted once the overall view is finalized.

**Fixes #1090**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: contributions/catalog/webapps/templates/contribute/contribution_details.html
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/contribution_details.html:1]: image
Here, the orange line is shown only when the list is displayed. Please see the design.

File: contributions/catalog/webapps/templates/contribute/contribution_details.html [hunk 1]
```diff
@@ -511,6 +511,86 @@ <h4 class="modal-title" id="confirmDeleteTitle">Delete Contribution</h4>
             document.getElementById("myDropdown").classList.toggle("show");
         }
     })
+    /*
+    // Custom dropdown menu functions
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/contribution_details.html:515]: Please remove the commented-out code. Thanks.

Comment (minump) [reply contributions/catalog/webapps/templates/contribute/contribution_details.html:515]: "The javascript needed for the full customization is commented out. The other related styling classes is also present in the style.css file. These can be deleted once the overall view is finalized."
Without that javascript code, the full customization is not acheivable.

Comment (minump) [reply contributions/catalog/webapps/templates/contribute/contribution_details.html]: removed commented out javascript and extra stylings

File: contributions/catalog/webapps/templates/contribute/contribution_details.html
Comment (sandeep-ps) [reply contributions/catalog/webapps/templates/contribute/contribution_details.html:1]: Resolving. Thanks.

**Comment by sandeep-ps** [comment]
There is another select control when choosing the privacy level for Talent. It could be updated as part of this PR if it's straightforward. Please let me know. Thanks.

**Comment by minump** [comment]
> There is another select control when choosing the privacy level for Talent. It could be updated as part of this PR if it's straightforward. Please let me know. Thanks.

Didnt update that as it is not really a dropdown menu as shown in zeplin.
If we are updating that select control in the Talent, should the upward and downward arrow be two orange carets as well?
Screen Shot 2023-03-15 at 2 07 05 PM

**Comment by sandeep-ps** [comment]
> > There is another select control when choosing the privacy level for Talent. It could be updated as part of this PR if it's straightforward. Please let me know. Thanks.
> 
> Didnt update that as it is not really a dropdown menu as shown in zeplin. If we are updating that select control in the Talent, should the upward and downward arrow be two orange carets as well? Screen Shot 2023-03-15 at 2 07 05 PM

Okay. Sounds good.

**Review by sandeep-ps — APPROVED** [review]
Thanks. The changes look fine. Approving.

**Pull request merged**